.github/CODEOWNERS

```diff
@@ -3,14 +3,14 @@
 #  @department-of-veterans-affairs/backend-review-group and @department-of-veterans-affairs/va-api-engineers will be requested for
 # review when someone opens a pull request.
 
+# Global default code owners
+* @department-of-veterans-affairs/backend-review-group
```

Comment:
What this will do is assign `backend-review-group` to every new file added to the vets-api repo, which isn't desired. We want new files to be owned by a team (and `backend-review-group` will own files as a fall-back if there are no teams to own them).

We have a GitHub Action that looks at PRs for new files created and then checks CODEOWNERS to make sure it's got a code owner. If there is no codeowner, a bot sends a message in the PR. If we had this line, no files would ever be flagged as needing an owner because `backend-review-group` would always be the owner. I hope that makes sense 😃

Reply:
Yes, I agree with you description; that makes perfect sense. It seems like we should go change the logic of the bot, not enforce adding the group to every line. Could you point me to the bot? I wouldn't mind investigating.

Reply:
The bot would be able to raise a flag `if codeowners.count == 1 && codeowners.first == backend-review-group`? That would indicate there was only one owner, the default owner  `backend-review-group` from our entry at the top of `CODEOWNERS`. You could still encourage people to claim ownership and avoid the excess noise.

Reply:
The GH Action: https://github.com/department-of-veterans-affairs/vets-api/blob/master/.github/workflows/check_codeowners.yml
Which calls this script: https://github.com/department-of-veterans-affairs/vets-api/blob/master/.github/scripts/check_codeowners.sh

Reply:
@mathisto The CODEOWNERS app, which name is actually Platform Atlas,  you are referring to will not fix this file going forward. Initially, maybe 50% of files were owned, which was very daunting on the backend-review-team. Now that this CODEOWNERS file has been updated and there are checks to ensure new files are owned, there is no need for Platform Atlas to modify this file